lib/ethpy/ethpy/base/retry_utils_test.py

```diff
@@ -1,16 +1,22 @@
 """Test executing transactions."""
 
+from typing import cast
+
 import pytest
-from ethpy.base import retry_call
-from ethpy.hyperdrive.interface import HyperdriveReadInterface
+from eth_typing import URI
+from ethpy.base import initialize_web3_with_http_provider, retry_call
+from ethpy.test_fixtures import DeployedHyperdrivePool
+from web3 import HTTPProvider
 
 
 @pytest.mark.anvil
-def test_retry_call_success(hyperdrive_read_interface: HyperdriveReadInterface, caplog: pytest.LogCaptureFixture):
+def test_retry_call_success(local_hyperdrive_pool: DeployedHyperdrivePool, caplog: pytest.LogCaptureFixture):
```

Comment:
We should be able to use the interface in this test here to reduce some code, since it's now back in ethpy

Reply:
I decided to leave this alone bc the interface has a lot of overhead that we aren't using in this test.